pkg/plugins/plugins_test.go

```diff
@@ -21,6 +21,17 @@ func TestFindPlugin(t *testing.T) {
 	assert.Equal(t, path, nd.Join("tkn-test"))
 }
 
+func TestFindPluginNoExec(t *testing.T) {
```

Comment:
Consider adding positive test case:
`tkn-test` is alongside `tkn` but not in `$PATH`